parcels/kernel/basekernel.py

```diff
@@ -4,7 +4,6 @@
 import numpy.ctypeslib as npct
```

Comment:
In this file, I see and like the simplification - no code generation-related issues anymore.